### ISSUE TITLE
Option to manually set CC values?

### ISSUE THREAD

**Issue by kraffslol** [sbycrosz/react-native-credit-card-input#6]
Hello!
I'm trying to set the card values manually but I can't seem to figure out how to do it (If even possible).

I tried adding
```JS
values={{expiry: "11/11"}}
``` 
prop to the CreditCardInput component but it seems like the values prop doesn't get set.

Is there a way to set the initial values yet?

**Comment by sbycrosz** [comment]
Hey @kraffslol 

I just released 0.2.5 which has a method that you could access by refs 
e.g. `this.refs.CCInput.setValues({ expiry: "11/11" })`

Let me know if this covers your use case!
- Sam

**Comment by kraffslol** [comment]
Yes, thats exactly what I was looking for! Thank you.

**Comment by smerkousdavid** [comment]
I know this issue has been closed but I found a quick solution (hopefully this prop is actually added to the component) to add default values without having to use a ref.

```
class DefaultCreditCardInput extends CreditCardInput {
  constructor(props) {
    super(props);

    this.state = {
      ...this.state,
      values: props.values
    };
  }
}
....
<DefaultCreditCardInput
   values={{number: '123512312', name: 'Charles Ramrodding'}}
....
```

Hopefully this helps (there is a PR to fix this issue #105)
